lightning/src/ln/msgs.rs

```diff
@@ -622,6 +622,8 @@ mod fuzzy_internal_msgs {
 	#[derive(Clone)]
 	pub(crate) struct FinalOnionHopData {
 		pub(crate) payment_secret: PaymentSecret,
+		/// The total value, in msat, of the payment as received by the ultimate recipient.
+		/// Message serialization may panic if this value is more than 21 million Bitcoin.
```

Comment:
nit: s/may panic/will panic and below as well

Reply:
I generally prefer to avoid committing to that. While we *currently* panic, I don't want a user relying on that specific behavior.